Sofa/Component/Topology/Container/Dynamic/src/sofa/component/topology/container/dynamic/EdgeSetTopologyContainer.cpp

```diff
@@ -72,8 +74,6 @@ void EdgeSetTopologyContainer::init()
         }
     }
 
-    PointSetTopologyContainer::init();
```

Comment:
hum, no I think this is needed to init all topology containers.
What is missing is probably:
in PoitnSetTopologyContainer
core::topology::TopologyContainer::init();